### PR TITLE
Parse explicit lifetime dependence specifiers in SIL

### PR DESCRIPTION
**Pull request by meg-gupta**

### CODE REVIEW

**Review by atrick — APPROVED** [review]
LGTM thanks!

**Comment by meg-gupta** [comment]
@swift-ci smoke test

File: lib/Parse/ParseDecl.cpp [hunk 1]
```diff
@@ -5048,13 +5048,13 @@ ParserStatus Parser::parseTypeAttribute(TypeOrCustomAttr &result,
...
 getLifetimeDependenceKind(const Token &T) {
-  if (T.isContextualKeyword("_copy")) {
+  if (T.isContextualKeyword("_copy") || T.isContextualKeyword("_inherit")) {
     return LifetimeDependenceKind::Copy;
   }
   if (T.isContextualKeyword("_consume")) {
     return LifetimeDependenceKind::Consume;
   }
-  if (T.isContextualKeyword("_borrow")) {
+  if (T.isContextualKeyword("_borrow") || T.isContextualKeyword("_scope")) {
```
Comment (ahoppen) [inline lib/Parse/ParseDecl.cpp:5057]: Do we need to support `_inherit` and `_scope` in the new parser as well?

Comment (meg-gupta) [reply lib/Parse/ParseDecl.cpp:5057]: This spelling is for SIL tests only. I don't think this code path goes through the new parser.

**Comment by meg-gupta** [comment]
@swift-ci smoke test linux platform

File: lib/Parse/ParseDecl.cpp (hunk 1 above)
Comment (ahoppen) [reply lib/Parse/ParseDecl.cpp:5057]: Ah, I didn’t read this properly. Yes, looks like SIL-only which doesn’t need support in the new parser.

**Pull request merged**